includes/MirahezeMagicHooks.php

```diff
@@ -29,14 +29,30 @@ public static function onCreateWikiCreation( $DBname ) {
 	}
 
 	public static function onCreateWikiDeletion( $dbw, $wiki ) {
-		if ( file_exists( "/mnt/mediawiki-static/{$wiki}" ) ) {
-			Shell::command( '/bin/rm', '-rf', "/mnt/mediawiki-static/{$wiki}" )->execute();
+		global $wgEchoSharedTrackingDB;
+
+		$dbw = wfGetDB( DB_MASTER, [], $wgEchoSharedTrackingDB );
+		
+		if ( $dbw->selectRowCount( 'echo_unread_wikis', '*', [ 'euw_wiki' => $wiki ] ) > 0 ) {
```

Comment:
Why are we running a query to check if we should run a query, when running the one query is sufficient?

Reply:
Fixed